### PR TITLE
ART-1906: Use separate signing advisory for each release

### PR DESCRIPTION
**Pull request by vfreex**
Signing advisories will be read from group.yml instead of hardcoded 54765.

### CODE REVIEW

File: pipeline-scripts/buildlib.groovy [hunk 1]
```diff
@@ -1483,7 +1483,7 @@ def getChanges(yamlData) {
  *
  * }
  */
-def buildBuildingPlashet(version, release, el_major, include_embargoed, auto_signing_advisory=54765) {
+def buildBuildingPlashet(version, release, el_major, include_embargoed, auto_signing_advisory) {
```
Comment (sosiouxme) [inline pipeline-scripts/buildlib.groovy:1486]: if there is no signing advisory in the group.yml `auto_signing_advisory` will be empty and signing won't occur. i'm thinking of 4.2 which still builds... we can either set it to `54765` if not given, or we can throw an error at this point (and update 4.2 `group.yml` to use `54765`). I think I'd prefer continuing to default if not specified.

**Review by sosiouxme — CHANGES_REQUESTED** [review]
minor issue, otherwise LGTM

File: pipeline-scripts/buildlib.groovy (hunk 1 above)
Comment (vfreex) [reply pipeline-scripts/buildlib.groovy]: Right, let's continue to use the default signing advisory if not specified.

**Review by sosiouxme — APPROVED** [review]
Thanks!

**Pull request merged**